src/utils/events/event-manager.js

```diff
@@ -1,13 +1,15 @@
-import {Manager} from 'hammerjs';
+import WheelInput from './wheel-input';
+import MoveInput from './move-input';
 
-import {
+const Manager = (typeof window === 'undefined' || typeof document === 'undefined') ?
+  ManagerMock : require('hammerjs').Manager;
```

Comment:
As for the `isBrowser` test, and `src/controllers` generally, that folder should be cleaned up to bring it more in-line with `react-map-gl`...and the [`globals.js`](https://github.com/uber/deck.gl/blob/master/src/controllers/globals.js) file there seems to be a near-duplicate of [`src/utils/globals.js`](https://github.com/uber/deck.gl/blob/master/src/lib/utils/globals.js)...thoughts on this @ibgreen @shaojingli?

For now, I can use that `isBrowser`.